Emby.Server.Implementations/Library/CoreResolutionIgnoreRule.cs

```diff
@@ -15,16 +16,19 @@ public class CoreResolutionIgnoreRule : IResolverIgnoreRule
     {
         private readonly ILibraryManager _libraryManager;
         private readonly IServerApplicationPaths _serverApplicationPaths;
+	private readonly IServerConfigurationManager _configurationManager;
```

Comment:
This project uses spaces, not tabs.

```suggestion
        private readonly IServerConfigurationManager _configurationManager;
```

Reply:
I hate tabs too (no idea where they came from). :+1: